### PR TITLE
fix #212: [FR] Fix for keywords-only 'fr-verbe-flexion' template

### LINKED ISSUE #212: [FR] Fix for keywords-only 'fr-verbe-flexion' template
Commands:
```
$ python -m scripts --locale="fr" --raw --get-word="tange"
$ python -m scripts --locale="fr" --raw --get-word="engrave"
$ python -m scripts --locale="fr" --raw --get-word="dites"
$ python -m scripts --locale="fr" --raw --get-word="perlés"
```

Wikicode:
```
{{fr-verbe-flexion|grp=3|1=dire|imp.p.2p=oui|ind.p.2p=oui|ppfp=oui}}
```

Output:
```
Traceback (most recent call last):
  File "scripts/__main__.py", line 77, in <module>
    sys.exit(main(sys.argv[1:]))
  File "scripts/__main__.py", line 59, in main
    return get.main(args.locale, word=args.get_word, raw=args.raw_output)
  File "scripts/get.py", line 538, in main
    get_and_parse_word(word, locale, raw=raw)
  File "scripts/get.py", line 336, in get_and_parse_word
    details = parse_word(word, code, locale, force=True)
  File "scripts/get.py", line 409, in parse_word
    infinitive = clean(word, t.__str__(), locale)
  File "scripts/utils.py", line 282, in clean
    text = text.replace(tpl, transform(word, tpl[2:-2], locale))
  File "scripts/utils.py", line 348, in transform
    result: str = transform_apply(word, tpl, tuple(parts), locale)
  File "venv/lib/python3.8/site-packages/cachetools/decorators.py", line 22, in wrapper
    v = func(*args, **kwargs)
  File "scripts/utils.py", line 364, in transform_apply
    return eval(templates_multi[locale][tpl])  # type: ignore
  File "<string>", line 1, in <module>
IndexError: list index out of range
```

### PR DESCRIPTION
**Pull request by lasconic**
- [x] Fixes #212
- [x] Tests added/updated.
- [x] `./check.sh` run and eventual issues fixed.
- [x] Updated CONTRIBUTORS.md, if not already done in the past.

### CODE REVIEW

File: scripts/lang/fr/__init__.py [hunk 1]
```diff
@@ -855,6 +864,17 @@ def last_template_handler(template: Tuple[str, ...], locale: str) -> str:
...
         return term(phrase)
 
+    # Handle the {{fr-verbe-flexion}} template
+    if tpl == "fr-verbe-flexion":
+        phrase = ""
+        data = extract_keywords_from(parts)
+        if data["1"]:
+            phrase = data["1"]
+        else:
+            tag = [p for p in parts if "=" not in p]
```
Comment (BoboTiG) [inline scripts/lang/fr/__init__.py:874]: `parts` cannot contains strings with "=", those are removed by `extract_keywords_from()`.
You could simply reduce the whole thing to:
```python
data = extract_keywords_from(parts)
return data.get("1", parts[0])
```

**Comment by lasconic** [comment]
OK. It can be a single line


```python
extract_keywords_from(parts).get("1", parts[0] if parts else "")
```

You prefer to have it in `template_multi` or `last_template_handler ` ?

**Comment by BoboTiG** [comment]
👍 in `last_template_handler()`.

**Comment by lasconic** [comment]
OK done.

**Pull request merged**